.github/workflows/stress.yml

```diff
@@ -0,0 +1,21 @@
+name: Stress Test
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Keeping the `push:` key in for testing purposes -- will remove this right before merging.

The reason we need this is that `workflow_dispatch` events can only be triggered manually from the `master` branch.